central/image/service/http_handler_test.go

```diff
@@ -39,8 +175,27 @@ func TestHttpHandler_ServeHTTP(t *testing.T) {
 		t.Setenv(features.ScannerV4.EnvVar(), "true")
 		ctrl := gomock.NewController(t)
 		defer ctrl.Finish()
+
+		// mockSbom returns sample sbom
+		sbom := map[string]interface{}{
+			"SPDXID":      "SPDXRef-DOCUMENT",
+			"spdxVersion": "SPDX-2.3",
+			"creationInfo": map[string]interface{}{
+				"created": "2023-08-30T04:40:16Z",
+				"creators": []string{
+					"Organization: Uchiha Cortez",
+					"Tool: FOSSA v0.12.0",
+				},
+			},
+		}
+		sbomBytes, err := json.Marshal(sbom)
+		assert.NoError(t, err)
+
+		// initliaze mocks
 		mockEnricher := enricherMock.NewMockImageEnricher(ctrl)
+		mockSbom := scannerTypeMocks.NewMockSBOMer(ctrl)
```

Comment:
The `mockSbom` in these tests is not used and can be removed - currently the `GetSBOM()` method of the `fakeRegistryScanner` is used instead.